hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -90,35 +90,33 @@
 
   public HoodieDeltaStreamer(Config cfg, JavaSparkContext jssc) throws IOException {
     this(cfg, jssc, FSUtils.getFs(cfg.targetBasePath, jssc.hadoopConfiguration()),
-        getDefaultHiveConf(jssc.hadoopConfiguration()));
+        jssc.hadoopConfiguration(), null);
```

Comment:
can we not pass `null` as a sentinel.. its risky business.. would an empty properties/map work?

Reply:
totally understand

Reply:
unfortunately, the `null` was served as a flag in many places in the delta streamer, because the command-line tool will produce `null`. If we want to change this, it might need a code refactoring.

Reply:
got it..